lib/package.py

```diff
@@ -123,29 +131,23 @@ def _load(self):
                 "Failed to open %s's YAML descriptor" % self.name)
 
         self.name = self.package_data.get('name')
-        self.clone_url = self.package_data.get('clone_url', None)
-        self.download_source = self.package_data.get('download_source', None)
+        self.sources = self.package_data.get('sources', [])
 
-        # Most packages keep their version in a VERSION file
-        # or in the .spec file. For those that don't, we need
-        # a custom file and regex.
         version = self.package_data.get('version', {})
         self.version_file_regex = (version.get('file'),
                                    version.get('regex'))
 
-        # NOTE(maurosr): Unfortunately some of the packages we build
-        # depend on a gziped file which changes according to the build
-        # version so we need to get that name somehow, grep the
-        # specfile would be uglier imho.
+        # (ltadeu): These are all being kept for compatibility reasons. They are
```

Comment:
Unrelated to this patch: I don't see the point of leaving the author's name in the comments. If we really need to find out who wrote them, git blame can be used.